src/main/java/gov/loc/repository/bagit/verify/QuickVerifier.java

```diff
@@ -33,9 +33,21 @@ private QuickVerifier(){
    * @return true if the bag can be quickly verified
    */
   public static boolean canQuickVerify(final Bag bag){
+    boolean payloadInfoExists = false;
+    
+    if(bag.getPayloadByteCount() != null && bag.getPayloadFileCount() != null){
+      logger.debug("Found payload byte and file count, using that instead of payload-oxum");
+      //TODO check if it matches payload-oxum, and if not issue warning?
```

Comment:
:+1: on a warning - I was just looking to see what would happen if both are present or if they are and do not match. I would use the bagit.txt fields in preference but issue a validation error or possibly just a linter warning if Payload-Oxum doesn't match the more controlled fields.